Apps/WebClient/src/ClientApp/app/router.ts

```diff
@@ -1,63 +1,107 @@
 import Vue from 'vue'
 import VueRouter from 'vue-router'
-import HomeComponent from './views/home.vue'
-import RegistrationComponent from './views/registration.vue'
-import LandingComponent from './views/landing.vue'
-import ImmunizationsComponent from './views/immunizations.vue'
-import NotFoundComponent from './views/errors/notFound.vue'
-import LogoutComponent from './views/logout.vue'
-import UnauthorizedComponent from './views/errors/unauthorized.vue'
+import store from './store'
+import HomeComponent from '../app/views/home.vue'
+import RegistrationComponent from '../app/views/registration.vue'
+import LandingComponent from '../app/views/landing.vue'
+import ImmunizationsComponent from '../app/views/immunizations.vue'
+import NotFoundComponent from '../app/views/errors/notFound.vue'
+import LoginComponent from '../app/views/login.vue'
+import LogoutComponent from '../app/views/logout.vue'
+import UnauthorizedComponent from '../app/views/errors/unauthorized.vue'
+import TempAuthComponent from '../app/views/auth.vue'
 
 Vue.use(VueRouter)
 
+function load(componentPath: string) {
+    // '@' is aliased to src/components, so we can't use it with our structure
+    return () => import(`./views/${componentPath}`)
+}
+
 const routes = [
-    { path: '/', component: LandingComponent },
-    { path: '/registration', component: RegistrationComponent },
     {
+        path: '/',
+        component: LandingComponent,
+        meta: { requiresAuth: false }
+    },
+    {
+        path: '/registration',
+        component: RegistrationComponent,
+        meta: { requiresAuth: false }
+    },
+    {
+        name: 'home',
         path: '/home',
         component: HomeComponent,
         meta: { requiresAuth: true, roles: ['user'] },
         children: [
-            { path: '/immunizations', component: ImmunizationsComponent, meta: { requiresAuth: true, roles: ['user'] } }
+            {
+                path: '/immunizations',
+                component: ImmunizationsComponent,
+                meta: { requiresAuth: true, roles: ['user'] }
+            }
         ]
     },
+    {
+        path: '/auth',
+        component: TempAuthComponent,
```

Comment:
Is this one still needed?

Reply:
wasn't Stephen still using it for debugging?